packages/cli/src/modules/lint/alpha.ts

```diff
@@ -76,6 +76,10 @@ export default createCliPlugin({
           '--since <ref>',
           'Only lint packages that changed since the specified ref',
         );
+        command.option(
+          '--max-warnings <number>',
+          'Fail if more than this number of warnings. -1 allows warnings. (default: 0)',
```

Comment:
Wasn't the idea that the default should be -1 both here and in the other lint command?

Reply:
ooops, thx! I've updated all the descriptions